### ISSUE TITLE
Inspector build failed in release build 3.1.3

### ISSUE THREAD

**Issue by mrclauss** [isar-community/isar#7]
### Steps to Reproduce
```

Target dart2js failed: ProcessException: Process exited abnormally with exit code 1:
../../../../../.pub-cache/hosted/pub.dev/google_fonts-3.0.1/lib/src/google_fonts_base.dart:14:1:
Error: 'AssetManifest' is imported from both 'package:flutter/src/services/asset_manifest.dart' and 'package:google_fonts/src/asset_manifest.dart'.
import 'asset_manifest.dart';
^^^^^^^^^^^^^
../../../../../.pub-cache/hosted/pub.dev/go_router-5.2.4/lib/src/information_provider.dart:38:34:
Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
      location: routeInformation.location!,
                                 ^
../../../../../.pub-cache/hosted/pub.dev/go_router-5.2.4/lib/src/parser.dart:59:59:
Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
      initialMatches = matcher.findMatch(routeInformation.location!,
                                                          ^
../../../../../.pub-cache/hosted/pub.dev/go_router-5.2.4/lib/src/parser.dart:71:40:
Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
            Uri.parse(routeInformation.location!),
                                       ^
../../../../../.pub-cache/hosted/pub.dev/go_router-5.2.4/lib/src/parser.dart:72:69:
Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
            MatcherError('no routes for location', routeInformation.location!)
                                                                    ^
../../../../../.pub-cache/hosted/pub.dev/google_fonts-3.0.1/lib/src/google_fonts_base.dart:32:31:
Error: 'AssetManifest' is imported from both 'package:flutter/src/services/asset_manifest.dart' and 'package:google_fonts/src/asset_manifest.dart'.
AssetManifest assetManifest = AssetManifest();
                              ^^^^^^^^^^^^^
Error: Compilation failed.
  Command: /opt/hostedtoolcache/flutter/stable-3.19.2-x64/bin/cache/dart-sdk/bin/dart --disable-dart-dev /opt/hostedtoolcache/flutter/stable-3.19.2-x64/bin/cache/dart-sdk/bin/snapshots/dart2js.dart.snapshot --platform-binaries=/opt/hostedtoolcache/flutter/stable-3.19.2-x64/bin/cache/flutter_web_sdk/kernel --invoker=flutter_tool -Ddart.vm.product=true -DFLUTTER_WEB_AUTO_DETECT=true -DFLUTTER_WEB_CANVASKIT_URL=https://www.gstatic.com/flutter-canvaskit/04817c99c9fd4956f27505204f7e344335810aed/ --native-null-assertions --no-source-maps -o /home/runner/work/isar/isar/packages/isar_inspector/.dart_tool/flutter_build/8e17adfc7100c984a207b7cd34e33a25/app.dill --packages=.dart_tool/package_config.json --cfe-only /home/runner/work/isar/isar/packages/isar_inspector/.dart_tool/flutter_build/8e17adfc7100c984a207b7cd34e33a25/main.dart
#0      RunResult.throwException (package:flutter_tools/src/base/process.dart:127:5)
#1      _DefaultProcessUtils.run (package:flutter_tools/src/base/process.dart:326:19)
<asynchronous suspension>
#2      Dart2JSTarget.build (package:flutter_tools/src/build_system/targets/web.dart:188:5)
<asynchronous suspension>
#3      _BuildInstance._invokeInternal (package:flutter_tools/src/build_system/build_system.dart:861:9)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:518:21)
<asynchronous suspension>
#5      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:799:32)
<asynchronous suspension>
#6      Future.wait.<anonymous closure> (dart:async/future.dart:518:21)
<asynchronous suspension>
#7      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:799:32)
<asynchronous suspension>
#8      FlutterBuildSystem.build (package:flutter_tools/src/build_system/build_system.dart:628:16)
<asynchronous suspension>
#9      WebBuilder.buildWeb (package:flutter_tools/src/web/compile.dart:93:34)
<asynchronous suspension>
#10     BuildWebCommand.runCommand (package:flutter_tools/src/commands/build_web.dart:200:5)
<asynchronous suspension>
#11     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:[138](https://github.com/isar-community/isar/actions/runs/8123688279/job/22204397354#step:4:139)9:27)
<asynchronous suspension>
#12     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:[153](https://github.com/isar-community/isar/actions/runs/8123688279/job/22204397354#step:4:154):19)
<asynchronous suspension>
#13     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#14     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:355:9)
<asynchronous suspension>
#15     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#16     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:295:5)
<asynchronous suspension>
#17     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:119:9)
<asynchronous suspension>
#18     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#19     main (package:flutter_tools/executable.dart:90:3)
<asynchronous suspension>
Compiling lib/main.dart for the Web...                             13.5s
Error: Failed to compile application for the Web.
Error: Process completed with exit code 1.

```
### Code sample

```dart
Provide a few simple lines of code to show your problem.
```

### Details

 - Platform: iPhone 13 Pro, Galaxy S7, x86 Android Emulator on Windows etc.
 - Flutter version: [e.g. 3.0.0]
 - Isar version: [e.g. 2.5.0]

---

 - [ ] I searched for similar issues already
 - [ ] I filled the details section with the exact device model and version
 - [ ] I am able to provide a reproducible example

**Comment by vicenterusso** [comment]
If I remember correctly I fixed this error building the inspector with a lower flutter version. try 3.7.12 or lower

**Comment by mrclauss** [comment]
The CI actions always use a recent version. I fixed it by upgrading to google fonts 6.1.0 (since 6.2.0 has a different bug right now)
